Specs/Core/VRTheWorldTerrainProviderSpec.js

```diff
@@ -244,8 +244,8 @@ describe('Core/VRTheWorldTerrainProvider', function() {
         it('provides HeightmapTerrainData', function() {
             var baseUrl = 'made/up/url';
 
-            Resource._Implementations.createImage = function(url, crossOrigin, deferred) {
-                expect(url.indexOf('.tif?cesium=true')).toBeGreaterThanOrEqualTo(0);
+            Resource._Implementations.createImage = function(request, crossOrigin, deferred) {
+                expect(request.url.indexOf('.tif?cesium=true')).toBeGreaterThanOrEqualTo(0);
 
                 // Just return any old image.
                 Resource._DefaultImplementations.createImage(imageUrl, crossOrigin, deferred);
```

Comment:
Shouldn't all the `Resource._DefaultImplementations.createImage` calls in the tests now take `request`? There are a bunch spread throughout the tests.

Reply:
You're right. And all of the spies need to be changed too. For example this one:
https://github.com/AnalyticalGraphicsInc/cesium/blob/24aa1f7b1a2ef404cc88ebea4020521d7d945f87/Specs/Scene/UrlTemplateImageryProviderSpec.js#L113-L116